src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4476,102 +4456,118 @@ namespace SimAirServingZones {
 					ZoneSA = 0.0;
 					ZonePA = 0.0;
 					Ep = 1.0;
-					if ( FinalZoneSizing( CtrlZoneNum ).ZoneSecondaryRecirculation > 0.0 ) { // multi-path system
+					if ( TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZoneSecondaryRecirculation > 0.0 ) { // multi-path system
 						//Vpz: "Primary" supply air from main air handler served by an oa mixer
-						ZonePA = FinalZoneSizing( CtrlZoneNum ).DesCoolVolFlow;
+						ZonePA = TermUnitFinalZoneSizing( TermUnitSizingIndex ).DesCoolVolFlow;
 						//Vdz: "Discharge" supply air delivered to zone by terminal unit
 						ZoneSA = max( TermUnitSizing( TermUnitSizingIndex ).AirVolFlow, ZonePA );
```

Comment:
maybe this instance of TermUnitSizing should now be TermUnitFinalZoneSizing, line 4463 SimAirServingZones.cc

Reply:
TermUnitSizing holds values that are set in the terminal units, e.g. [here for PIU](https://github.com/NREL/EnergyPlus/blob/VAVDesignMinFlow6327/src/EnergyPlus/PoweredInductionUnits.cc#L1084-L1091). The values *should be* based on TermUnitFinalZoneSizing with any scaling.